src/jsMain/kotlin/io/beatmaps/user/main.kt

```diff
@@ -135,137 +148,226 @@ class ProfilePage : RComponent<ProfilePageProps, ProfilePageState>() {
                                     }
 
-                                    +if (following) "Unfollow" else "Follow"
-                                }
-                            }
-
-                            +"Maps: ${it.totalMaps}, Upvotes: ${it.totalUpvotes}, Downvotes: ${it.totalDownvotes}"
-                            br { }
-                            +"Average BPM: ${it.avgBpm}, Average Score: ${it.avgScore}%, "
-                            +"Average Duration: ${it.avgDuration.formatTime()}"
-                            it.diffStats?.let { ds ->
-                                br { }
-                                b {
-                                    +"Easy: "
-                                }
-                                +"${ds.easy}"
-                                b {
-                                    +", Normal: "
-                                }
-                                +"${ds.normal}"
-                                b {
-                                    +", Hard: "
-                                }
-                                +"${ds.hard}"
-                                b {
-                                    +", Expert: "
-                                }
-                                +"${ds.expert}"
-                                b {
-                                    +", Expert+: "
+                                    if (following) {
+                                        i("fas fa-user-minus") { }
+                                        +"Unfollow"
+                                    } else {
+                                        i("fas fa-user-plus") { }
+                                        +"Follow"
+                                    }
                                 }
-                                +"${ds.expertPlus}"
                             }
                         }
-                        /*img("100 Maps", "https://cdn.discordapp.com/avatars/98334361564246016/01ade7513a63215bb7937d217b766da3.png", classes = "rounded-circle mx-2") {
-                            attrs.width = "75"
-                            attrs.height = "75"
+                    }
+                }
+            }
+            div("col-md-8 mb-3 position-relative") {
+                div("card user-info") {
+                    div("card-body") {
+                        span {
+                            textToContent((state.userDetail?.description?.take(500) ?: ""))
                         }
-                        img("100 Maps", "https://cdn.discordapp.com/avatars/98334361564246016/01ade7513a63215bb7937d217b766da3.png", classes = "rounded-circle mx-2") {
-                            attrs.width = "75"
-                            attrs.height = "75"
+                        state.userDetail?.stats?.let {
+                            if (it.totalMaps != 0) {
+                                table("table table-dark") {
```

Comment:
Not a huge fan of the table. Mostly the bottom border, looks slightly better with table-borderless. Can possibly roll a custom style.

Also someone's going to complain about losing avg bpm/duration even if they're not that useful.

Reply:
I'll try to fit average bpm and duration in there, though no one has complained so far, and I don't know anyone who uses it.